### ISSUE TITLE
Overhaul "Donations"

### ISSUE THREAD

**Issue by efstajas** [drips-network/app#1299]
The are some glaring problems with the concept of "Donations" on Drips right now, which are mostly result of independently evolving functionality in different places over time. Let's revisit this concept holistically and clean things up a bit.

- **Merge conflated concepts of "Streams" and "Continuous Donations"**: Currently, you can go to a Drip List and create a "Continuous Donation", and this shows up on the Funds screen as a "Stream". There's also a "Create stream" button here which opens effectively the same modal as when you click Support -> Continuous Donation on a Project or Drip List. We should fix the terminology by referring to "Streams" as "Continuous Donations" all across the UX. Also, the Streams section on the Funds screen should probably be replaced with a Donations section that shows ongoing Continuous Donations alongside a history of One-Time Donations made by the currently-connected wallet. 
- **Missing Support Card on Address Profiles**. The Support Card appears on Drip Lists and Projects, but not on Address profiles. That's weird and there's no reason for it.
- **Unable to create a Continuous Donation directly to a Project**: You can still only establish a Continuous Donation to Drip Lists (and addresses). There's no reason for that — it should be possible to continuously donate to projects too.
- **No date pickers on the Continuous Donation Creation Modal**. Setting start & end dates is super awkward right now because you need to manually enter date strings in UTC. We already have a Date Picker component — let's use it.
- **Can't top-up a token together with creating a Continuous Donation**: When you create a new Continuous Donation, you should be able to immediately top-up its token in the same TX. This is possible on the protocol level and would be useful, so let's allow it in the UX.
- **Different & separate Continuous Donation configurator in Create Drip List flow**: There are two completely different modals used for when you configure a Continuous Donation within the Create Drip List flow, and when you later create a Continuous Donation on an existing Drip List. The one in the Create Drip List flow also doesn't allow setting start & end dates for the donation. There's no reason for these to be different.

![Image](https://github.com/user-attachments/assets/7b7e0c4a-18c6-43f3-8f82-3786a1958fc2)

**Comment by brandonhaslegs** [comment]
@efstajas Yeah it's here https://www.figma.com/file/vyI7f996JF8zwhnXwAwXdC?node-id=12300:3560&locale=en&type=design
